Tools/Create-Azure-Sentinel-Solution/input/Solution_CohesitySecurity.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Name": "CohesitySecurity",
+    "Author": "Microsoft - support@microsoft.com",
+    "Description": "TBD",
```

Comment:
@yinghuang123, Can you please add the description if decided, thanks.

Reply:
The file was removed